website/themes/docsy/userguide/package.json

```diff
@@ -18,7 +18,7 @@
   "homepage": "https://github.com/bep/tech-doc-hugo#readme",
   "dependencies": {},
   "devDependencies": {
-    "autoprefixer": "^9.5.0",
```

Comment:
Why is the change to website and themes folder being made? 
This doesn't seem to be relevant to the self-assessment and I'd suggest discarding this change

Reply:
I think this happened when I synced our repo to the cncf main branch. Sorry for the confusion, I can checkout this file to an older commit. Looks similar to this dependabot [commit](https://github.com/cncf/tag-security/pull/1172/commits/96b4f6bbc7bd9fb052df5241a7b959d27f645fc5), but the autoprefixer commit was likely squashed

Reply:
I reverted the website folder back in commit [7c1c8ae](https://github.com/dhauss/tag-security/commit/abb8f8e076bd44da45685ebffffaf429b6d4ab1a), I believe this should fix the issue but let me know if it didn't, admittedly my first time doing this particular git move